src/Cli/Microsoft.DotNet.Cli.Utils/BuiltInCommand.cs

```diff
@@ -57,7 +57,7 @@ public CommandResult Execute()
 
                     if (!string.IsNullOrEmpty(_workingDirectory))
                     {
-                        _environment.SetWorkingDirectory(_workingDirectory);
+                        _environment.SetWorkingDirectory(_workingDirectory!);
```

Comment:
*! Justification*: `string.IsNullOrEmpty` check

Reply:
I've decided to switch the `string.IsNullOrEmpty` check to explicit null and Length check to avoid the usage of `!`.